Dockerfile

```diff
@@ -60,18 +62,18 @@ RUN chmod +x /odooboot
 # Get Odoo code
 WORKDIR /opt
 RUN set -x; \
-  git clone --depth 1 https://github.com/oca/ocb.git -b 10.0 odoo \
-  && rm -rf odoo/.git
+  git clone --depth 1 https://github.com/oca/ocb.git -b 10.0 ocb \
+  && rm -rf ocb/.git
```

Comment:
This is not a change we want to implement as it breaks our standard practices of server operation.